src/pages/patientView/clinicalInformation/PatientViewPageStore.ts

```diff
@@ -311,14 +340,36 @@ export class PatientViewPageStore {
     });
 
     readonly mutationMolecularProfileId = remoteData({
-        await: () => [this.molecularProfilesInStudy],
+        await: () => [this.mutationMolecularProfile],
         invoke: async () => {
-            const profile = findMutationMolecularProfile(
-                this.molecularProfilesInStudy,
-                this.studyId
+            if (this.mutationMolecularProfile.result) {
+                return this.mutationMolecularProfile.result.molecularProfileId;
+            } else {
+                return undefined;
+            }
+        },
+    });
+
+    readonly structuralVariantProfile = remoteData({
+        await: () => [this.studyIdToMolecularProfiles],
+        invoke: async () => {
+            const structuralVariantProfiles = getFilteredMolecularProfilesByAlterationType(
+                this.studyIdToMolecularProfiles.result,
+                AlterationTypeConstants.STRUCTURAL_VARIANT,
+                [DataTypeConstants.SV, DataTypeConstants.FUSION]
             );
-            if (profile) {
-                return profile.molecularProfileId;
+            if (structuralVariantProfiles.length > 0) {
+                return structuralVariantProfiles[0];
+            }
+            return undefined;
+        },
+    });
+
+    readonly structuralVariantProfileId = remoteData({
```

Comment:
seems kind silly to make a whole mobxpromise wrapper just to reference the id.  any calling code will still have to check the status of this, so we don't save really anything